src/main/java/com/mattmalec/pterodactyl4j/client/entities/impl/CreateDatabaseImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.mattmalec.pterodactyl4j.client.entities.impl;
+
+import com.mattmalec.pterodactyl4j.client.entities.Backup;
+import com.mattmalec.pterodactyl4j.client.entities.ClientServer;
+import com.mattmalec.pterodactyl4j.client.entities.Database;
+import com.mattmalec.pterodactyl4j.client.managers.BackupAction;
+import com.mattmalec.pterodactyl4j.client.managers.DatabaseAction;
+import com.mattmalec.pterodactyl4j.entities.P4J;
+import com.mattmalec.pterodactyl4j.requests.PteroActionImpl;
+import com.mattmalec.pterodactyl4j.requests.Route;
+import com.mattmalec.pterodactyl4j.utils.Checks;
+import okhttp3.RequestBody;
+import org.json.JSONObject;
+
+import java.util.function.Supplier;
+
+public class CreateDatabaseImpl extends PteroActionImpl<Database> implements DatabaseAction {
```

Comment:
Likewise from the previous comment, these should follow the client naming conventions

```suggestion
public class CreateDatabaseImpl extends PteroActionImpl<ClientDatabase> implements ClientDatabaseAction {
```